pygments/lexers/ezhil.py

```diff
@@ -69,7 +69,7 @@ def analyse_text(text):
         decent amount of Tamil-characters, it's this language. This assumption
         is obviously horribly off if someone uses string literals in tamil
         in another language."""
-        if len(re.findall('[\u0b80-\u0bff]')) > 10:
+        if len(re.findall(r'[\u0b80-\u0bff]', text)) > 10:
```

Comment:
Oops, I missed that.

Reply:
Me too, but that's what ~friends~ tests are for :)